charts/zitadel/templates/pdb.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.pdb.enabled }}
+{{- if .Values.pdb.apiVersion }}
+apiVersion: {{ .Values.pdb.apiVersion }}
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "zitadel.fullname" . }}
+  labels:
+    {{- include "zitadel.labels" . | nindent 4 }}
```

Comment:
labels annotations as well like in the ingress template

Reply:
Nice one! missed that